source/Calamari.Terraform/ApplyTerraformConvention.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using Calamari.Deployment;
+using Calamari.Integration.FileSystem;
+using Calamari.Integration.Substitutions;
+using Newtonsoft.Json.Linq;
+
+namespace Calamari.Terraform
+{
+    public class ApplyTerraformConvention : TerraformConvention
+    {
+        public ApplyTerraformConvention(ICalamariFileSystem fileSystem, IFileSubstituter fileSubstituter) : base(fileSystem, fileSubstituter)
+        {
+        }
+
+        protected override void Execute(RunningDeployment deployment, Dictionary<string, string> environmentVariables)
+        {
+            IEnumerable<(string, JToken)> OutputVariables(string result)
```

Comment:
Does this need to be an inner function? feels like its contributing to the `Execute` method feeling bloated

Reply:
fixed